graphein/ml/datasets/pdb_data.py

```diff
@@ -1879,7 +1972,7 @@ def export_pdbs(
             defaults to ``None``.
         :type splits: Optional[List[str]], optional
         :param max_num_chains_per_pdb_code: Maximum number of chains
-            to collate into a matching PDB file.
+            to collate into a matching PDB file, defaults to ``-1``.
```

Comment:
Is this designed to throw an error if not provided by the user or behave like a wildcard selector?

Reply:
Currently, this argument is used as such:

```python
chains = (
                        chains
                        if max_num_chains_per_pdb_code == -1
                        else chains[:max_num_chains_per_pdb_code]
                    )
```

Reply:
So if the user doesn't specify this argument's value, then the default behavior is to select all chains associated with a given PDB complex.